component.js

```diff
@@ -142,6 +142,12 @@ function factory (options) {
     }
 
     var Component = React.createClass(componentObject);
+    
+    // If there is a globally or locally set decorator, apply it to the class
+    var decorator = options.decorate || _decorate;
```

Comment:
So it seems, too early in the morning :coffee:

But yeah, we're also shadowing options, maybe a better name is in place for the last one?